tests/CMakeLists.txt

```diff
@@ -41,7 +41,14 @@ if( ALGEBRA_PLUGINS_INCLUDE_VC )
    algebra_add_test( vc
       "vc/vc_cmath.cpp" "vc/vc_vc.cpp"
       LINK_LIBRARIES GTest::gtest_main algebra::tests_common
-                     algebra::vc_cmath algebra::vc_vc )
+                      algebra::vc_storage algebra::vc_cmath algebra::vc_vc )
```

Comment:
This line should not need a change. Just linking against the "frontent" libraries should be enough. :thinking:

Reply:
Done